aria/orchestrator/workflows/core/events_handler.py

```diff
@@ -147,7 +147,8 @@ def _update_node_state_if_necessary(ctx, is_transitional=False):
     # and also will *never* be the type name
```

Comment:
I tried to squash all these commits to one.. could you please check if it is fine now ?